dashboard/db/migrate/20170626160941_add_storage_app_to_channel_tokens.rb

```diff
@@ -0,0 +1,5 @@
+class AddStorageAppToChannelTokens < ActiveRecord::Migration[5.0]
+  def change
+    add_column :channel_tokens, :storage_app_id, :integer, after: :channel
```

Comment:
The existing names for some of these things (not your fault) make me sad. The `storage_apps` table should really be called the `channels` table, although I'm not sure we'll ever get around to renaming it given its size.

A better name for `storage_app_id` would be `channel_id`, with a comment indicating that it is the unencrypted channel id whereas `channel` is the encrypted channel id.